plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSplitSource.java

```diff
@@ -72,7 +90,9 @@ public HudiSplitSource(
             ScheduledExecutorService splitLoaderExecutorService,
             int maxSplitsPerSecond,
             int maxOutstandingSplits,
-            List<String> partitions)
+            List<String> partitions,
```

Comment:
I'll skip the review on the dynamic filter changes which I've provided reviews in #12 .